src/components/atoms/Title/Title.ts

```diff
@@ -1,5 +1,5 @@
 import styled from 'styled-components';
-import { Theme } from '../../../app/theme';
+import { Theme } from '../../../libs/facets/theme';
```

Comment:
I'm not convinced `Theme` definition should live nor in facets nor in libs.... could it be kept in components?
Could our components decently avoid depending on our libs? Might not be practical tho'

Reply:
Also, prefer an absolute import over a relative one.

Reply:
If we go full lib style, we could have 
- /libs/facet <- /components depends on this lib
- /libs/dismoiFacet <- depends on /libs/facet
- /libs/lmelFacet <- depends on /libs/facet

but for a few files and 2 themes, I thougth I’ll keep it simple for now …

Reply:
relative import fixed